gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/controller/GraphControllerTest.java

```diff
@@ -107,18 +105,6 @@ void getGraphs_ReturnsGraphsAsList_whenSuccessful() throws Exception {
         assertEquals(200, getGraphsResponse.getResponse().getStatus());
     }
 
-//    @Test
-//    void authEndpointShouldReturn200StatusAndTokenWhenValidUsernameAndPassword() throws Exception {
-//        final String authRequest = "{\"username\":\"javainuse\",\"password\":\"password\"}";
```

Comment:
Should this be disabled or deleted?

Reply:
Deleted as rest does not handle authentication anymore so authentication is tested inside sidecar.